### DISCUSSION TITLE
Custom logo for the signin page.

### DISCUSSION THREAD

**Discussion by pnzj-dev** [nextauthjs/next-auth#4409] [Help]
Hi everybody,

I've tried to follow the documentation to setup the custom logo for the signin page. But it's not working, does someone as a working example of this particular setting ? Thanks for the help.

` theme: {
        colorScheme: "auto", // "auto" | "dark" | "light"
        brandColor: "#D902D7", // Hex color code
        logo: "", // Absolute URL to image
    },`

in [...nextauth].ts

**Comment by muhaimincs** [comment]
just put the absolute url to the `logo` props
